rocky/assets/js/toggleSwitch.js

```diff
@@ -0,0 +1,51 @@
+import { onDomReady } from "../js/imports/utils.js";
+
+onDomReady(initToggleSwitches);
+
+function initToggleSwitches() {
+  // Switches get stitches ;-)
+  let toggle_switches = document.querySelectorAll(".toggle-switch");
+
+  for (let i = 0; i < toggle_switches.length; i++) {
+    let options = toggle_switches[i].querySelectorAll(".toggle-switch-button");
+
+    for (let j = 0; j < options.length; j++) {
+      let option = options[j];
+
+      // Hide all elements linked to toggle switch options.
+      document
+        .getElementById(option.getAttribute("data-target-id"))
+        .classList.add("hidden");
+
+      // Add click listener to switch options.
+      option.addEventListener("click", (event) => {
+        toggle(event.target, options);
+      });
+    }
+
+    // Initially show first option contents from toggle-switch.
+    document
+      .getElementById(options[0].getAttribute("data-target-id"))
```

Comment:
Dont asume 0 is the 'current active' one when initing. the developer might have added an aria-current to one of the switches.

Reply:
Good point...